### PR TITLE
Added description field

### PR DESCRIPTION
**Pull request by mord4z**
Added a new field, description, the user can pass a parameter with
detailed data about the transition and save in state log table

### CODE REVIEW

**Comment by hzy** [comment]
So, after discussion in #12, I'm happy to have this change merged. A couple of things need to happen first

- Revert changes to the migration files, and create new migrations to add the description field
- Fix tests
- Add a test or two

cc @Alexkane

**Comment by Alexkane** [comment]
That's great news, thank you.

**Comment by hzy** [comment]
@Alexkane are you able to make those changes? I currently don't have any time to work on this project

**Comment by mord4z** [comment]
Can I help make those changes?

**Comment by hzy** [comment]
is it possible to define a transition as

```python
     @fsm_log_by
     @transition(field=state, source='submitted', target='published')
     def publish(self, by=None, description=None):
         pass
```

then when calling the transition do `article.publish(description="I felt like publishing this right now")`?

File: django_fsm_log/south_migrations/0002_auto__add_field_statelog_description.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        u'contenttypes.contenttype': {
+            'Meta': {'ordering': "('name',)", 'unique_together': "(('app_label', 'model'),)", 'object_name': 'ContentType', 'db_table': "'django_content_type'"},
+            'app_label': ('django.db.models.fields.CharField', [], {'max_length': '100'}),
+            u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
+            'model': ('django.db.models.fields.CharField', [], {'max_length': '100'}),
+            'name': ('django.db.models.fields.CharField', [], {'max_length': '100'})
+        },
+        u'django_fsm_log.statelog': {
+            'Meta': {'object_name': 'StateLog'},
+            'by': ('django.db.models.fields.related.ForeignKey', [], {'to': u"orm['auth.Usuario']", 'null': 'True', 'blank': 'True'}),
```
Comment (hzy) [inline django_fsm_log/south_migrations/0002_auto__add_field_statelog_description.py]: use the tricks from the initial migration to get the `by` field to hook up correctly https://github.com/gizmag/django-fsm-log/blob/master/django_fsm_log/south_migrations/0001_initial.py#L46

**Comment by JoeConyers** [comment]
Any update on this merge?

**Comment by Alexander-Attar** [comment]
Was this ever merged in?

**Comment by blueyed** [comment]
description was merged in #67 - not yet released.

**Comment by intellineers** [comment]
Any update on when this gets released? or is it save to use the unreleased version?

**Comment by blueyed** [comment]
@intellineers 
Should be save, but it is not guaranteed really, since migrations are involved, and might change.
But I don't expect that.

It seems like @ticosax had taken over maintenance lately, but might currently not being interested anymore, so a new maintainer might be needed.

**Pull request closed without merging**